Dockerfile

```diff
@@ -1,3 +1,20 @@
+FROM gradle:7-jdk11-alpine AS build
+
+COPY --chown=gradle:gradle . /build
+WORKDIR /build
+
+ARG VERSION_TAG
```

Comment:
What happens when you run a local Docker build without specifying a VERSION_TAG from the command line?

Reply:
Good question, looks like the lower level defaults to 0.1.0 upon receiving a blank `-Prelease.forceVersion=` option—same as it does if none were provided. The wrapper script would then be broken though. I've implemented your above suggestion to work around the problem and verified that local builds work with or without VERSION_TAG provided as a build arg now